### DISCUSSION TITLE
is there a forum or a community on discord or any other platform for coqui tts related things

### DISCUSSION THREAD

**Discussion by king-dahmanus** [coqui-ai/TTS#1636] [General Q&A]
hi. Is there a community for sharing models, asking for help, etc on discord preferably, or any other platform? Please give me all the options available. The kind of help I want isn't that code related, so github issues are out of the question. I'd like to recrute someone who can teach me how to train models for all supported architectures with  a collab notebook collection or something, please note that I'm a blind guy, so  spectrogram analysis is impossible for me. I need text and audio, nothing else. Thanks.

**Comment by erogol** [comment]
Gitter room (matrix room), GitHub discussion is waiting for you. Links are on the README
